src/machine/machine-varlink.c

```diff
@@ -728,3 +729,99 @@ int vl_method_map_to(sd_varlink *link, sd_json_variant *parameters, sd_varlink_m
 
         return sd_varlink_reply(link, v);
 }
+
+typedef struct MachineMountParameters {
+        const char *name;
+        PidRef pidref;
+        char *src;
+        char *dest;
```

Comment:
as in the other commit, make this const char* and use json_dispatch_const_path() to parse